packages/kilt-did/src/DidDetails/LightDidDetails.utils.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2018-2021 BOTLabs GmbH.
+ *
+ * This source code is licensed under the BSD 4-Clause "Original" license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+import { encode as cborEncode, decode as cborDecode } from 'cbor'
+import type { LightDidDetailsCreationOpts } from './LightDidDetails'
+
+const ENCRYPTION_KEY_MAP_KEY = 'encryptionKey'
+const SERVICES_MAP_KEY = 'services'
```

Comment:
S: we might be able to save some bytes, by using shorter keys here

Reply:
Fixed in https://github.com/KILTprotocol/sdk-js/pull/408/commits/ed4d9c3448b4bfc329e87bdd5930b88f88287e64.